### PR TITLE
Napa

### PR DESCRIPTION
**Pull request by boomerwv1**

### CODE REVIEW

File: locations/spiders/napa.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                "city": data["address"]["addressLocality"],
+                "state": state,
+                "postcode": data["address"]["postalCode"],
+                "country": data["address"]["addressCountry"],
+                "phone": data.get("telephone"),
+                "lat": data["geo"]["latitude"],
+                "lon": data["geo"]["longitude"],
+            }
+
+            yield GeojsonPointItem(**properties)
```
Comment (iandees) [inline locations/spiders/napa.py:56]:
```suggestion
            yield Feature(**properties)
```

File: locations/spiders/napa.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            "city": data["address"]["addressLocality"],
+            "state": state,
+            "postcode": data["address"]["postalCode"],
+            "country": data["address"]["addressCountry"],
+            "phone": data.get("telephone"),
+            "lat": data["geo"]["latitude"],
+            "lon": data["geo"]["longitude"],
+        }
+
+        yield GeojsonPointItem(**properties)
```
Comment (iandees) [inline locations/spiders/napa.py:87]:
```suggestion
        yield Feature(**properties)
```

**Review by iandees — COMMENTED** [review]
We switched `GeojsonPointItem` to `Feature` in #4597

File: locations/spiders/napa.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+            try:
+                state = data["address"]["addressRegion"]
+            except:
+                state = ""
+
+            properties = {
+                "ref": data["url"],
+                "name": data["name"],
+                "addr_full": data["address"]["streetAddress"],
```
Comment (iandees) [inline locations/spiders/napa.py:46]:
```suggestion
                "street_address": data["address"]["streetAddress"],
```

File: locations/spiders/napa.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+        try:
+            state = data["address"]["addressRegion"]
+        except:
+            state = ""
+
+        properties = {
+            "ref": data["url"],
+            "name": data["name"],
+            "addr_full": data["address"]["streetAddress"],
```
Comment (iandees) [inline locations/spiders/napa.py:77]:
```suggestion
            "street_address": data["address"]["streetAddress"],
```

**Review by iandees — COMMENTED** [review]
It looks like they're blocking us. Can you try setting a User Agent?

**Comment by Cj-Malone** [comment]
I'm getting blocked from GB with a browser user agent. But I think it's cloudflare.

**Comment by boomerwv1** [comment]
This one ran for me locally and returned about 5000 locations before I pushed to here.

**Comment by iandees** [comment]
Yep, looks like they're using Cloudflare.

**Pull request merged**